modules/ppcp-button/src/Assets/SmartButton.php

```diff
@@ -1318,8 +1318,22 @@ public function script_data(): array {
 			'should_handle_shipping_in_paypal'        => $this->should_handle_shipping_in_paypal && ! $this->is_checkout(),
 			'needShipping'                            => $this->need_shipping(),
 			'vaultingEnabled'                         => $this->settings->has( 'vault_enabled' ) && $this->settings->get( 'vault_enabled' ),
+			'productType'                             => null,
+			'manualRenewalEnabled'                    => '',
```

Comment:
Maybe should be `false`. It may be converted into string when passing to JS, but I think it would be better to keep it as a normal boolean here (like all other values) to avoid confusion.